UPGRADE.md

```diff
@@ -2,6 +2,11 @@
 
 ## dev-develop
 
+### Rename comment_count property to commentCount
+
+The `comment_count` property of serialized threads was renamed to `commentCount`.
```

Comment:
this should not change can we check the jms serializer config in the bundle:

Sulu overwrite it here:

https://github.com/sulu/sulu/blob/87fdd3500d5999548acc07ca4b4df9842a9956f4/src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php#L129-L138

Reply:
looks like the bundle does not prepend any `property_naming` config. how should we deal with this?

Reply:
in general i would prefer `commentCount` to `comment_count`.

Reply:
The bundle should not need todo it was the sulu/sulu does it.
Its  not about whats better its just a bc break which we don't know where it comes from and that was worried me.